### PR TITLE
Students bug fix

### PR DESCRIPTION
**Pull request by MichaelValkiv**
More information about class in table. Fixed row height. Changed class buttons margins. Added loading spinner while loading information.

### CODE REVIEW

File: src/app/admin-panel/admin-panel/students/students.component.html [hunk 1]
```diff
@@ -1,18 +1,18 @@
 <p-fieldset legend="Активні класи" [toggleable]="true" [collapsed]="false">
   <span *ngFor="let  class_ of classes">
-    <button *ngIf="class_.isActive == true" type="button" pButton icon="pi pi-folder" label="&nbsp;&nbsp;{{class_.className}}" title="{{class_.id}}" (click)="loadStudents(class_.id)" class="ui-button-raised classButton"></button>
+    <button *ngIf="class_.isActive == true" type="button" pButton icon="pi pi-folder" label="&nbsp;&nbsp;{{class_.className}}" title="{{class_.id}}" (click)="loadStudents(class_.id); selectedClassNameHendler(class_.className + ' класу')" class="ui-button-raised classButton"></button>
```
Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/students/students.component.html]: JFYI no need to compare boolean properties in conditions

Comment (dmitryy90) [inline src/app/admin-panel/admin-panel/students/students.component.html]: typo: handler

Comment (MichaelValkiv) [reply src/app/admin-panel/admin-panel/students/students.component.html]: Changed.

Comment (MichaelValkiv) [reply src/app/admin-panel/admin-panel/students/students.component.html]: Fixed.

**Pull request merged**